Tournament_Calculator.py

```diff
@@ -0,0 +1,323 @@
+                        pair = (group[0], next_group[0])
+                        if len(next_group) == 3:
+                            pair2 = (next_group[1], next_group[2])
+                            while pair2 in previous_pairings or pair[::-1] in previous_pairings:
+                                random.shuffle(next_group)
+                                pair = (group[0], next_group[0])
+                            if len(group) == 3:
+                                pair3 = ((group[1], group[2]))
+                                while pair3 in previous_pairings or pair[::-1] in previous_pairings:
+                                    random.shuffle(group)
+                                    pair = (group[0], next_group[0])
+                    odd_player = group.pop()
+                    paired_player = next_group.pop(0)
+                    pairings.append((odd_player, paired_player))
+                    previous_pairings.add((odd_player, paired_player))
+                    break
+                next_group_index += 1
+            else:
+                # If no suitable pairing found, give the odd player a bye (free win)
+                odd_player = group.pop()
+                results[odd_player] = 1
+        random.shuffle(group)  # Randomize player order within the group
+        for j in range(0, len(group), 2):
+            pair = (group[j], group[j+1])
+            while pair in previous_pairings or pair[::-1] in previous_pairings:
+                random.shuffle(group)  # Shuffle players again to try different pairings
+                pair = (group[j], group[j+1])
+            pairings.append(pair)
+            previous_pairings.add(pair)
+    
+    return pairings, previous_pairings
+
+def simulate_round(pairings):
+    # Simulate games for each pairing
+    results = {}
+    for pair in pairings:
+        # Simulate game between pair[0] and pair[1]
+        # Update results based on game outcome (win, loss, draw)
+        # For simplicity, let's assume random results
+        Coin_Flip = random.randint(1,2)
+        if Coin_Flip == 1:
+            Hero1 = Player_List.get(pair[0])
+            Hero2 = Player_List.get(pair[1])
+            Weight = Matchups[Hero1]["First"][Hero2]
+        elif Coin_Flip == 2:
+            Hero1 = Player_List.get(pair[0])
+            Hero2 = Player_List.get(pair[1])
+            Weight = Matchups[Hero1]["Second"][Hero2]
+        Deceider = round(random.uniform(0,1),2)
+        results[pair[0]] = 'win' if Deceider <= Weight else 'loss' if Deceider > Weight else 'draw'
+        results[pair[1]] = 'win' if results[pair[0]] == 'loss' else 'loss' if results[pair[0]] == 'win' else 'draw'
+    return results
+
+
+previous_pairings = set()
+results = {key: 0 for key in Player_List.keys()}
+Tie_Breakers = {key: 0 for key in Player_List.keys()}
+for round_num in range(int(Rounds)):
+    round_num+1
+    """print("\n" "Round", round_num+1)"""

```

Comment:
What's up with these strings? Looks like you maybe had temp code that you wanted to turn on and off, normally we'll just comment lines out instead of changing them to strings since that doesn't work in a lot of languages